form/unit_testing.rst

```diff
@@ -54,9 +54,9 @@ The simplest ``TypeTestCase`` implementation looks like the following::
                 'test2' => 'test2',
             ];
 
-            $formData = new TestObject();
+            $objectToCompare = new TestObject();
```

Comment:
What about `$data` or `$model`?

Reply:
I'd rather go for $model. $data looks kind of broad to me as it could be confused with the data which is submitted in the form.
What do you think?

Reply:
Fine for me, `$model` also matches the term "model data" that is used inside the Form component.